### PR TITLE
dep cleanup fixes

### PR DESCRIPTION
**Pull request by jeremygaither**

### CODE REVIEW

**Comment by jeremygaither** [comment]
Actually, I’m not sure this is the right way. Maybe `vendor/` should be in gitignore? But adding the source and branch properties to Gopkg means that file cannot be shared between forks. If everything was using `$GOPATH/src/` then changing git remotes would suffice, but dep doesn’t seem to handle things that way.

File: Gopkg.toml [hunk 1]
```diff
@@ -26,8 +26,8 @@
 
 
 [[constraint]]
-  branch = "ss-master"
-  name = "github.com/shipstation/go-teamcity-sdk"
+  name = "github.com/cvbarros/go-teamcity-sdk"
```
Comment (ss-remygreinhofer) [inline Gopkg.toml:29]: Why would you add the repo from cvbarros back? it does not work with our version of the provider anymore?

File: Gopkg.toml [hunk 2]
```diff
@@ -26,8 +26,8 @@
...
+  source = "../go-teamcity-sdk"
```
Comment (ss-remygreinhofer) [inline Gopkg.toml:30]: The `source` has to be relative to your $GOPATH, it cannot be anywhere you want.

**Review by ss-remygreinhofer — CHANGES_REQUESTED** [review]

File: Gopkg.toml (hunk 2 above)
Comment (ss-remygreinhofer) [reply Gopkg.toml:30]: Scratch that, it is the repo on github..., nothing to do with the file system.

**Comment by jeremygaither** [comment]
Not the right way to solve this problem. Go build and dep seem to have different rules, which seems very odd... further research needed.

**Pull request closed without merging**